Tests/secrets_white_list.json

```diff
@@ -1883,7 +1884,7 @@
       "384:mzawcK1gOUqJMpMhE5yOaF0O9TwFExCZkUf6H+PknKV+rXkrsH/CxEdubVAjMPW4:AiMVWcK9AhrDNEf"
     ],
     "emails": [
-      "demisto-sdk.git@1.2.18",
+      "demisto-sdk@finally-fixed-3.9",
```

Comment:
this can be removed (as it part of the requirement.txt file which should be ignored in the `secrets` command